implementations/rust/ockam/ockam/Cargo.toml

```diff
@@ -106,6 +106,7 @@ tracing = { version = "0.1", default-features = false }
 ockam_vault = { path = "../ockam_vault", version = "^0.133.0" }
 rand_xorshift = "0.3"
 serde_json = "1.0"
+tokio = "1.43.1"
```

Comment:
Why was it required?

Reply:
To wait a bit in the tests and check if messages were received or not.